### PR TITLE
Feature: Hello Microverse

### PR DESCRIPTION
**Pull request by RayhanTabase**
## Feature: Hello Microverse
In this feature we:
- Add index.html
- Add style.css
- Add .gitignore
- Add linters; .github/workflows/linters.yml
- Update README.md

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*test.md
```
Comment (dasileker) [inline .gitignore:1]: - [ ] Also you should have added the linters so you can add the `node_modules` to the `gitignore` file [click here to add linters](https://github.com/microverseinc/linters-config/tree/master/html-css).

MMM

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @RayhanTabase,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (RayhanTabase) [reply .gitignore:1]: Thank you for the feedback, I have reviewed your requirements and will proceed to install node and the linters locally and then add the node_modules to the .gitignore file.

**Review by mikethreels — APPROVED** [review]
Hi Salim, 

Good job implementing the requested changes!
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**